proto/vtgate.proto

```diff
@@ -75,7 +75,8 @@ message Session {
     topodata.TabletAlias tablet_alias = 3;
     // reserved connection if a dedicated connection is needed
     int64 reserved_id = 4;
-    bool vindex_only = 5;
+    // read_only is true if the session has only executed read queries.
+    bool read_only = 5;
```

Comment:
Not super familiar with Vitess protos, is it kosher to reuse a value with a different label?

Reply:
If the name isn't persisted anywhere I believe it should be ok. Are there any instances where this is serialized to something like json?